subjects/hello-world/README.md

```diff
@@ -0,0 +1,43 @@
+# hello-world
+
+| Complexity | Executable | Goal |
+|------|------|------|
+| Low | `greet` | Create a CLI Program that outputs a greeting.
+
+This subject is purely aimed to trying out the validation workflow.
+
+## Feature
+
+### Greet
+
+The program `greet` *may* have an argument to specify who to greet.
+
+```console
+$ greet Bob
+Hello Bob.
+```
+
+If no person to greet is specified, `greet` the World.
+
+```console
+$ greet
+Hello World.
+```
+
+## Additional targets
+
+??? success "Packaging"
```

Comment:
Exclamation marks are not supported as GitHub markdown.

Reply:
Project kind of switched to visualization through mkdocs, see https://github.com/open-subjects/open-subjects/pull/16#issuecomment-800662300